### PR TITLE
Support long strings.

### PR DESCRIPTION
**Pull request by oryonatan**
Now we can have multiline arrays, and strings are edited in a separate VC.

### CODE REVIEW

File: FBTweak/_FBEditableTweakTableViewCell.m [hunk 1]
```diff
@@ -258,9 +261,11 @@ - (void)_updateMode:(_FBTweakTableViewCellMode)mode
     self.imageView.hidden = NO;
   } else if (_mode == _FBTweakTableViewCellModeDate) {
     _switch.hidden = YES;
-    _textField.hidden = NO;
-    _textField.enabled = NO;
+    _textField.hidden = YES;
```
Comment (barakwei) [inline FBTweak/_FBEditableTweakTableViewCell.m:266]: does this mean that the date is not shown?

Comment (oryonatan) [reply FBTweak/_FBEditableTweakTableViewCell.m:266]: No this looks like this
image

File: FBTweak/_FBEditableTweakTableViewCell.m [hunk 2]
```diff
@@ -321,14 +326,14 @@ - (void)_updateValue:(FBTweakValue)value primary:(BOOL)primary write:(BOOL)write
     }
   } else if (_mode == _FBTweakTableViewCellModeString) {
     if (primary) {
-      _textField.text = value;
+      self.detailTextLabel.text = [value description];
```
Comment (barakwei) [inline FBTweak/_FBEditableTweakTableViewCell.m]: value is NSString, no? why do you need the description?

File: FBTweak/_FBEditableTweakTableViewCell.m (hunk 1 above)
Comment (barakwei) [reply FBTweak/_FBEditableTweakTableViewCell.m:266]: these are readonly! send a pic for editable tweaks

File: FBTweak/_FBEditableTweakTableViewCell.m (hunk 2 above)
Comment (oryonatan) [reply FBTweak/_FBEditableTweakTableViewCell.m]: fixed.

File: FBTweak/_FBEditableTweakTableViewCell.m (hunk 1 above)
Comment (oryonatan) [reply FBTweak/_FBEditableTweakTableViewCell.m:266]: oops here you go
image

Comment (barakwei) [reply FBTweak/_FBEditableTweakTableViewCell.m:266]: why is the date formatting different?

Comment (oryonatan) [reply FBTweak/_FBEditableTweakTableViewCell.m:266]: I think this format is nicer... I didn't touch FB implementation for the read-only though.

Comment (barakwei) [reply FBTweak/_FBEditableTweakTableViewCell.m:266]: I think they should be the same, don't care which format.  Note that their format shows the timezone which may be necessary.

Comment (oryonatan) [reply FBTweak/_FBEditableTweakTableViewCell.m:266]: It used to be like this when I used `[((NSDate *)value) description]`...

**Comment by oryonatan** [comment]
ראונד

File: FBTweak/FBTweaksDisplayUtils.h [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+// Created by Yonatan Oren.
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FBTweaksDisplayUtils : NSObject
+
+/// Convert a date to a short formatted string in the device's timezone.
++ (NSString *)getSimpleLocalFormattedDate:(NSDate *)date;
```
Comment (barakwei) [inline FBTweak/FBTweaksDisplayUtils.h]: This is not the best option. Either add a category on `NSDateFormatter` which returns the formatter (using a class property perhaps) and the implementation calls the converts using the formatter. Or you can have a category on `NSDate`.

**Comment by barakwei** [comment]
F&S

File: FBTweak/FBTweaksDisplayUtils.h (hunk 3 above)
Comment (barakwei) [reply FBTweak/FBTweaksDisplayUtils.h]: As discussed, remote `get` and let merge.

**Comment by oryonatan** [comment]
merge?

**Pull request merged**